kmk/modules/combos.py

```diff
@@ -223,6 +226,9 @@ def on_release(self, keyboard, key, int_coord):
                 self._key_buffer.append((int_coord, key, False))
                 key = None
 
+        # Reset on non-combo key up
+        if not self._matching:
+            self.reset(keyboard)
```

Comment:
```suggestion
            self.reset(keyboard)
            
```
add an empty line, see above.